test/test_bregman.jl

```diff
@@ -5,9 +5,9 @@ using LinearAlgebra
 
 N1 = 100
 N2 = div(N1, 2) + 5
-A = randn(N1, N2)
+A = randn(ComplexF32, N1, N2)
```

Comment:
If you want to add a case add an extra one do not modify an existing one especially if you are disabling options

Reply:
ok